src/main/java/com/wire/bots/hold/resource/v0/audit/EventsResource.java

```diff
@@ -16,17 +14,12 @@
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
-import java.io.IOException;
-import java.io.PrintWriter;
-import java.io.StringWriter;
-import java.util.List;
 import java.util.UUID;
 
 @Api
-@Path("/events/{conversationId}")
+@Path("/events/{conversationId}{noop: (/)?}{conversationDomain:([^/]+?)?}")
```

Comment:
`{noop: (/)?}` means that the `/` in-between values is optional if there is no latter value (domain), so acceptable calls are:
```
/conv/conversationId
/conv/conversationId/
/conv/conversationId/domain
```